autosomal_coverage.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import csv
+import argparse
+from glob import glob
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description="Script to calculate mean autosomal coverage after Sarek3.4.2 WGS analyses"
+    )
+    parser.add_argument(
+        "--analysis_dir",
+        required=False,
+        default="/proj/ngi2016001/nobackup/NGI/ANALYSIS",
+        help=f"Path to analysis folder. (default: %(default)s)",
+    )
+    parser.add_argument("--project", required=True, help="Project name")
+    args = parser.parse_args()
+    return args
+
+
+def find_reports(project_dir, analysis_dir, project):
+    """
+    Tries to find the Mosdepth reports in the specified directories.
+    In most cases the reports should be located in the project_dir.
+
+    Args:
+        project_dir (str): Path to the project directory.
+        analysis_dir (str): Path to the analysis directory.
+        project (str): Project name
+
+    Returns:
+        str: Path to the found Mosdepth reports directory.
+        str: Path to where output will be written.
+        None: If the reports cannot be found in either directory.
+    """
+
+    for dir_path in [project_dir, analysis_dir]:
```

Comment:
Are there cases where we have results written directly in the Analysis directory? Because with `analysis_dir`, `report_folder` becomes `/proj/ngi2016001/nobackup/NGI/ANALYSIS/results/reports/mosdepth` right?